packages/react/src/components/pickers/BasePicker.test.tsx

```diff
@@ -110,6 +110,32 @@ describe('BasePicker', () => {
     disabledTests: ['component-has-root-ref', 'component-handles-ref', 'has-top-level-file'],
   });
 
+  it('renders inline callout', () => {
+    jest.useFakeTimers();
```

Comment:
Can you add cleanup for this in `afterEach` to avoid potential interference with other tests?
```
if ((setTimeout as any).mock) {
  jest.useRealTimers();
}
```

Reply:
done :)